WordPress/gutenberg
# Framework: Fix server-registered fixtures script

This pull request seeks to resolve an issue where `npm run fixtures-regenerate` was not working.

There were a few contributing factors:

- As of #18986, the `get-server-blocks.php` script was run in the `tests-wordpress-phpunit` Docker image which, unlike previously, was not a WordPress installation folder structure expected by the script, where the script assumes either to be provided with an `ABSPATH` WordPress install path, or be able to reach out of its own plugin directory several levels up to the closest WordPress install.
- At least for me, while I might have expected that redirecting output within the Docker container would be reflected back into the host volume, this was not the case (`server-registered.json`). The script was changed so the output of the NPM script itself was redirected.
   - This required:
      - Running `wp-env run` directly, as otherwise `npm run wp-env run` would include undesirable extra redirected output
      - Changing/fixing `wp-env run` to avoid formatting output when piping/redirecting

**Testing instructions:**

Verify server-registered fixtures generation works as expected:

1. Remove existing file: `rm test/integration/full-content/server-registered.json`
2. Run `npm run fixtures-regenerate`
3. Note working directory is clean (regenerated without changes): `git status`
